csrc/multidevice/lower_communication.cpp

```diff
@@ -293,11 +293,9 @@ std::vector<Communication*> lowerCommunication(Expr* c) {
 }
 
 bool isLowerableToCommunication(Expr* expr) {
-  NVF_ERROR(
-      ir_utils::isTvOp(expr),
-      "Non-tv op is not supported yet: ",
-      expr->toString());
-  if (expr->isA<ReductionOp>()) {
+  if (!ir_utils::isTvOp(expr)) {
```

Comment:
Can you rebase? There are some changes like this that belong to the other PR.

Reply:
I thought I could merge the previous PR in before people started reviewing 😂